services/parser-database/connector.py

```diff
@@ -2,15 +2,56 @@
 services/databases"""
 import requests  # pylint: disable=import-error
 import logging
+import numpy as np  # pylint: disable=import-error
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import firestore
 import constants
 import env
 
 logging.basicConfig(level=logging.INFO)
 
+class Keywords:
+    """Class for storing articles.
+    """
+    def __init__(self, keyword):
+        self.keyword = keyword
+        self.articles_that_contain_keyword = {}
+
+    def to_dict(self):
+        article_dict = {
+            "keyword": self.number,
+            "contain": self.articles_that_contain_keyword,
+        }
+        return article_dict
+
+    @staticmethod
+    def from_dict(src):
+        self.number = src["number"]
+        self.id = src["id"]
+        self.content = src["keyword"]
+        self.wordCount = src["frequency"]
+
+cred = credentials.ApplicationDefault()
+firebase_admin.initialize_app(cred, {
+  'projectId': 'major-tom-285619',
```

Comment:
Is this needed, wouldn't the cloud run containers already run on this projectId and be defaulted on initialize_app?

Reply:
If it is not there I cannot test locally

Reply:
So this will be removed before merging?